### DISCUSSION TITLE
Feature Request: Change title of CSV download from timestamp-based to instead be name of chart being exported.

### DISCUSSION THREAD

**Discussion by cass-simplelegal** [apache/superset#21859] [Ideas]
Currently, when exporting a CSV, the title of the CSV file is saved as a timestamp-based title.

We've had users ask if we can make the CSV export a chart-based title instead. Is this doable?

See screenshot of how it currently works:

<img width="1391" alt="Screen Shot 2022-10-18 at 10 14 58 AM" src="https://user-images.githubusercontent.com/116036593/196465263-e687bb0d-37b8-45f6-83cd-7448a6856c8d.png">

**Comment by cchristofr** [comment]
+1 for this PR

**Comment by sfirke** [comment]
If anyone wants to take a shot at it, I think you'd modify this line: https://github.com/apache/superset/blob/0735680674416ff56403635d9ebeb335ff744144/superset/views/base.py#L187  I'm not immediately sure how to get the chart name for insertion into that string, too.  Might be nice to have the format include both, like chartname_timestamp.csv

**Comment by sfirke** [comment]
There is a PR for this from last month but it appears stalled: https://github.com/apache/superset/pull/27409 if anyone can help the contributor move past their problem it would be appreciated.

**Comment by rwibowo** [comment]
We have the same needed for this feature. Really helpful our operation if we need to download / save chart many times. If anyone can help the contributor move past their problem it would be appreciated.
